gtsam/base/GenericValue.h

```diff
@@ -29,13 +29,19 @@
 #include <iostream>
 #include <typeinfo> // operator typeid
 
+#ifdef _WIN32
+#define GENERICVALUE_VISIBILITY
+#else
+#define GENERICVALUE_VISIBILITY GTSAM_EXPORT
+#endif
+
```

Comment:
If this is only in one place, wouldn't it be better to just conditionally compile the function signature?

Reply:
This is for a class, I do not think it is good to ifdef on class definitions.

Reply:
Why is this workaround needed now, when Windows compiler was ok with `GTSAM_EXPORT` in the past? Might also be good to add a comment for future reference.

Reply:
@chrisbeall This will trigger a `LNKxxx` error on Windows, the reason is in https://github.com/borglab/gtsam/blob/develop/Using-GTSAM-EXPORT.md, I will add a ref to that :)

Reply:
Also, the sad truth is that while we used to have a Windows CI, it only runs a very small subset of the test suite, and this problem is caught by the Actions CI (which runs more tests). Anyway, we need someone with adequate Windows experience to solve this in a **final** way.

CC @dellaert @jlblancoc